modules/running-distributed-data-science-workloads-from-notebooks.adoc

```diff
@@ -48,110 +51,164 @@ If your cluster administrator does not define a default local queue, you must sp
-For information about how to create a project, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/using-data-science-projects_projects#creating-a-data-science-project_projects[Creating a data science project].
+* You can access a data science project that contains a workbench, and the workbench is running a default notebook image that contains the CodeFlare SDK, for example, the *Standard Data Science* notebook. 
+For information about projects and workbenches, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects[Working on data science projects].
 endif::[]
 ifdef::upstream[]
-* You have created a data science project that contains a workbench, and the workbench is running a default notebook image that contains the CodeFlare SDK, for example, the *Standard Data Science* notebook. 
-For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#creating-a-data-science-project_projects[Creating a data science project].
+* You can access a data science project that contains a workbench, and the workbench is running a default notebook image that contains the CodeFlare SDK, for example, the *Standard Data Science* notebook. 
+For information about projects and workbenches, see link:{odhdocshome}/working-on-data-science-projects[Working on data science projects].
 endif::[]
 
 * You have Admin access for the data science project.
 ** If you created the project, you automatically have Admin access. 
 ** If you did not create the project, your cluster administrator must give you Admin access.
 
+* You have logged in to {productname-long}.
 * You have launched your notebook server and logged in to your notebook editor.
 The examples in this procedure refer to the JupyterLab integrated development environment (IDE).
 
+ifndef::upstream[]
+* You have downloaded the demo notebooks provided by the CodeFlare SDK, as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/running-distributed-workloads_distributed-workloads#downloading-the-demo-notebooks-from-the-codeflare-sdk_distributed-workloads[Downloading the demo notebooks from the CodeFlare SDK].
+endif::[]
+ifdef::upstream[]
+* You have downloaded the demo notebooks provided by the CodeFlare SDK, as described in link:{odhdocshome}/working_with_distributed_workloads/#downloading-the-demo-notebooks-from-the-codeflare-sdk_distributed-workloads[Downloading the demo notebooks from the CodeFlare SDK].
+endif::[]
+
+
 .Procedure
-. Download the demo notebooks provided by the CodeFlare SDK.
-The demo notebooks provide guidelines for how to use the CodeFlare stack in your own notebooks.
+. In the JupyterLab interface, open the *demo-notebooks > guided-demos* folder. 
+. Double-click each notebook in turn to open the notebook.
+Notebook files are named `_<filename>_.ipynb`.
```

Comment:
Why should they take note of the filename? I only see one other notebook name called out - do we need this?

Reply:
Updated the text to clarify.